src/HomeWork7/HumanAndRobot/Сlothes/Clothes.java

```diff
@@ -0,0 +1,83 @@
+package HomeWork7.HumanAndRobot.Сlothes;
+
+public interface Clothes extends Boots, Blazer, Glasses, Hat, Jacket, Pants, Shirt, Socks {
```

Comment:
Clothes common interface for all these interfaces. Better create interface Clothes and classes for all these class. for example: class Boots implements Clothes